File: patterns/javascript-patterns/pattern104.js
// Pattern 104: ...

// *     *     *
//   *   *   *
//     * * *
// * * * * * * *
//     * * *
//   *   *   *
// *     *     *

let n = 7;
for (let i = 1; i <= n; i++) {
    let row = '';
    for (let j = 1; j <= n; j++) {
        if (j === i || j === n - i + 1 || i === Math.floor(n / 2) + 1 || j === Math.floor(n / 2) + 1) {
            row += '* ';
        } else {
            row += '  ';
        }
    }
    console.log(row);
}

File: patterns/python-patterns/pattern104.py
# Pattern 104: ...

# *     *     *
#   *   *   *
#     * * *
# * * * * * * *
#     * * *
#   *   *   *
# *     *     *

n = 7

for i in range(1, n + 1):
    for j in range(1, n + 1):
        if j == i or j == n - i + 1 or i == n // 2 + 1 or j == n // 2 + 1:
            print("* ", end="")
        else:
            print("  ", end="")
    print()
